async-embedded/src/task.rs

```diff
@@ -43,7 +41,7 @@ pub async fn r#yield() {
                 self.yielded = true;
                 // wake ourselves
                 cx.waker().wake_by_ref();
-                asm::sev();
+                unsafe { crate::signal_event_ready(); }
```

Comment:
I believe this is not required anymore after my fix: https://github.com/ferrous-systems/async-on-embedded/pull/2/files#diff-22ab3da1c5b4e44f95ba9b8000eda448R116
However, `sev` is required when task is woken from an interrupt context.

Reply:
Well, your fix included the `sev` call so I kept it